support/docs/partials/sig-link-html.hbs

```diff
@@ -0,0 +1,14 @@
+{{#if name}}{{#sig no-gfm=true ~}}
+{{{@depOpen}~}}
+<a href="{{{readmeLink}}}">
```

Comment:
I looked through their template code and this looks like the best way. Great work figuring this out, @greg5green!